pkg/asset/installconfig/azure/validation_test.go

```diff
@@ -255,6 +273,26 @@ func TestAzureInstallConfigValidation(t *testing.T) {
 			edits:    editFunctions{unsupportedHyperVGeneration},
 			errorMsg: `^\[controlPlane.platform.azure.type: Invalid value: "Standard_Dc4_v4": only disks with HyperVGeneration V1 are supported, compute\[0\].platform.azure.type: Invalid value: "Standard_Dc4_v4": only disks with HyperVGeneration V1 are supported\]$`,
 		},
+		{
+			name:     "Valid encryption at host",
+			edits:    editFunctions{validEncryptionAtHost},
+			errorMsg: "",
+		},
+		{
+			name:     "Valid disk encryption set",
+			edits:    editFunctions{validDiskEncryptionSet},
+			errorMsg: "",
+		},
+		{
+			name:     "Invalid encryption at host",
+			edits:    editFunctions{invalidEncryptionAtHost},
+			errorMsg: `^\S+`,
```

Comment:
We should probably check that the correct error message is being used

Reply:
Fixed.